src/pages/core/plugins/transfer-delegate.md

```
@@ -238,7 +238,7 @@ const { signature } = await transfer(umi, {
     collection: collectionItem,
   })
   .sendAndConfirm(umi);
-
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)